src/app/caesar-portal/caesar-home/caesar-home.module.ts

```diff
@@ -21,7 +21,7 @@ import { ProfileModule } from '../profile/profile.module';
         CaesarHomeComponent
     ],
     providers: [
-        GroupService,
+        // GroupService,
```

Comment:
1) do not comment code, if you don't need it remove it.
2) Are you sure no one except of group-area component using this service? As far as I remember it was moved out from the group are, maybe you should double check this - Can you confirm that it is not used outside group-are module?